### ISSUE TITLE
How to keep category tree up-to-date?

### ISSUE THREAD

**Issue by boobiq** [allegro/allegro-api#376]
Hello,

in our applicatio we have a mapping from our categories IDs to Allegro category IDs (specifically car parts categories). With some changes made recently, some of Allegro categories don't exist anymore (at least not under their former IDs).

For example, category 49186 (Motoryzacja / Części samochodowe / Ogrzewanie, wentylacja, klimatyzacja / Klimatyzacja / Chłodnice klimatyzacji (skraplacze)) doesn't exist anymore, after checking category tree, I was able to find it by the name under ID 258678.

Is there any changelog or any way to be informed about what exactly has changed, because our application stops working from day to day, since category doesn't exist anymore. We are looking for some way to automatically get new category IDs, because it can happen anytime and we would have to manually go category by category and search for its new ID.

In other words, what is the easiest way to find out new category IDs? :)

These are categories which stopped working since last change (all car parts categories, different category parents though):

49186 - Chłodnice klimatyzacji (skraplacze)
49190 - Przewody klimatyzacji
49195 - Dmuchawy
49197 - Nagrzewnice
147884 - Rezystory dmuchaw
251163 - Kompresory klimatyzacji
257313 - Flgi aluminiowe

Thank you.

**Comment by boobiq** [comment]
Hi,

yeah, I will try to refresh update category tree at some intervals, but sometimes it can be hard to find out the new number if the name of the category was changed or the name of the parent etc.

Maybe some small changelog (category 49186 changed to 123456 etc) would help :)

Thanks for the help.

**Comment by TomekKaczmarek** [comment]
Currently we do not plan to include such option in the nearest future. Although thank you for your suggestion - we will review it and it may be considered in our further development plans.
